src/Kunstmaan/SeoBundle/Repository/SeoRepository.php

```diff
@@ -13,25 +12,29 @@
  */
 class SeoRepository extends EntityRepository
 {
-
     /**
      * Find the seo information for the given entity
      *
-     * @param AbstractEntity $entity
+     * @param EntityInterface $entity
      *
      * @return Seo
      */
-    public function findFor(AbstractEntity $entity)
+    public function findFor(EntityInterface $entity)
     {
-        return $this->findOneBy(array('refId' => $entity->getId(), 'refEntityName' => ClassLookup::getClass($entity)));
+        /** @var Seo $seo */
+        $seo = $this->findOneBy([
```

Comment:
No need for extra variable.

Reply:
again, just for ease of readability and debugging, i've never liked calls within arguments and returns, the debugger wont store the values for inspection